File: app/Models/FeederStatusPoint.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class FeederStatusPoint extends Model
{
    public $timestamps = false;

    protected $fillable = [
        'type',
        'status_id',
        'feeder_id',
        'name',
    ];

    /**
     * Get the feeder that owns this status point.
     */
    public function feeder()
    {
        return $this->belongsTo(Feeder::class);
    }
}

File: go-backend-pln-monitoring/internal/model/feeder_status_point.go
package model

// FeederStatusPoint represents the feeder_status_point model.
type FeederStatusPoint struct {
	ID       uint   `json:"id" gorm:"primaryKey"`
	Type     string `json:"type"`
	StatusID string `json:"status_id"`
	FeederID uint   `json:"feeder_id"`
	Name     string `json:"name"`
}
